### PR TITLE
HW-54040 Fix long type and amount in list receipt u i

### PR DESCRIPTION
**Pull request by ymoskovko-epam**
Jira [HW-54040](https://jira.site1.hyperwallet.local/browse/HW-54040)

<p>
drawing
drawing
drawing
</p>
<p>
drawing
drawing
drawing
</p>

### CODE REVIEW

**Comment by ymoskovko-epam** [comment]
ui test should be updated as well. working on it...

File: Sources/Receipt/ReceiptTransactionTableViewCell.swift [hunk 1]
```diff
@@ -33,30 +33,119 @@ final class ReceiptTransactionTableViewCell: UITableViewCell {
...
+        defaultInit()
     }
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        defaultInit()
+    }
+
+    private func defaultInit() {
+        let line1StackView = UIStackView(frame: .zero)
```
Comment (jsaini-hw) [inline Sources/Receipt/ReceiptTransactionTableViewCell.swift]: rename line1StackView to titleStackView

File: Sources/Receipt/ReceiptTransactionTableViewCell.swift [hunk 2]
```diff
@@ -33,30 +33,119 @@ final class ReceiptTransactionTableViewCell: UITableViewCell {
...
+        setup(line1StackView, with: receiptTypeLabel, and: amountLabel)
+
+        let line2StackView = UIStackView(frame: .zero)
```
Comment (jsaini-hw) [inline Sources/Receipt/ReceiptTransactionTableViewCell.swift]: rename line2StackView to subTitleStackView

File: Sources/Receipt/ReceiptTransactionTableViewCell.swift (hunk 1 above)
Comment (ymoskovko-epam) [reply Sources/Receipt/ReceiptTransactionTableViewCell.swift]: fixed

File: Sources/Receipt/ReceiptTransactionTableViewCell.swift (hunk 2 above)
Comment (ymoskovko-epam) [reply Sources/Receipt/ReceiptTransactionTableViewCell.swift]: fixed

**Pull request merged**